src/marqo/tensor_search/tensor_search.py

```diff
@@ -1376,51 +1763,51 @@ def _vector_text_search(
                             "include": ["__chunks.__field_content", "__chunks.__field_name"]
                         }
                     },
```

Comment:
I think these changes just cook the indentation of the search, right? Can we keep the current indentation pattern, unless there's a beneficial reason to change it?

Reply:
Aren't they the same indentation?